ydb/library/yql/minikql/comp_nodes/mkql_grace_join.cpp

```diff
@@ -828,7 +834,13 @@ void DoCalculateWithSpilling(TComputationContext& ctx) {
     UpdateSpilling();
 
     if (!HasMemoryForProcessing()) {
+        const auto used = TlsAllocState->GetUsed();
```

Comment:
Logging here and in similar places of other files is excessive. This line of code may be executed on every row. I don't think we need millions of "yellow zone reached" messages in log. I believe it's enough to log only when we change state to spilling

Reply:
Ok